device/send.go

```diff
@@ -183,7 +188,8 @@ func (device *Device) SendHandshakeCookie(initiatingElem *QueueHandshakeElement)
 	var buff [MessageCookieReplySize]byte
 	writer := bytes.NewBuffer(buff[:0])
 	binary.Write(writer, binary.LittleEndian, reply)
-	device.net.bind.Send(writer.Bytes(), initiatingElem.endpoint)
+	// TODO(raggi): allocation could be avoided
```

Comment:
Here too.

Reply:
Removed in https://github.com/WireGuard/wireguard-go/pull/64/commits/23c2269626d980c95958a2b0fc166601a4a6c4dd.